packages/melody-streams/src/operators/attachEventWithElement.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2019 trivago N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { attachEvent } from './attachEvent';
+import { withElement } from './withElement';
+import { combineRefs } from './combineRefs';
+import { of } from 'rxjs';
+
+export const attachEventWithElement = (...events) => {
+    const eventsAndElement = [
+        attachEvent(...events),
+        withElement(el => of(el), null),
+    ];
+    const refHandler = combineRefs(
+        ...eventsAndElement.map(([handler]) => handler)
+    );
+    const streams = eventsAndElement.map(([_, stream]) => stream);
+    return [refHandler, ...streams];
```

Comment:
If we keep this API design we could use `streams[0], streams[1]` here instead of the spread operator for more explicitness, smaller size and better performance.

Reply:
Same goes for the other usages